lib/content-services/src/lib/document-list/data/share-datatable-adapter.ts

```diff
@@ -70,6 +70,10 @@ export class ShareDataTableAdapter implements DataTableAdapter {
         this.allowDropFiles = allowDropFiles;
     }
 
+    getType(_row: DataRow, col: DataColumn): string {
```

Comment:
what's the reason behind this method? you provide a useless parameter `_row` just to get the column type? maybe worth calling the method `getColumnType` and using only the `col` param?